Content-Controls/Replace-picture-content-control-with-chart/.NET/Replace-picture-content-control-with-chart/Program.cs

```diff
@@ -0,0 +1,145 @@
+﻿using Syncfusion.DocIO.DLS;
+using Syncfusion.DocIO;
+using Syncfusion.DocIORenderer;
+using Syncfusion.OfficeChart;
+
+
+// Open the file as stream.
+using (FileStream docStream = new FileStream(Path.GetFullPath(@"../../../Data/PictureContentControl_InputDocument.docx"), FileMode.Open))
+{
+    // Load file stream into Word document.
+    using (WordDocument document = new WordDocument(docStream, Syncfusion.DocIO.FormatType.Automatic))
+    {
+        string[] propertyNames = { "ContentControlProperties.Title", "ContentControlProperties.Type" };
+        string[] propertyValues = { "Chart", "ContentControlType.Picture" };
+        // Find BlockContentControl by given properties
+        BlockContentControl contentControl = document.FindItemByProperty(EntityType.BlockContentControl, "ContentControlProperties.Title", "Chart") as BlockContentControl;
```

Comment:
Cross check and correct the code properly to achieve the requirement.

Reply:
modified sir